src/main/webapp/pages/errors/oauth_error.html

```diff
@@ -4,7 +4,7 @@
     <title th:text="#{error.oauth.title}">_OAUTH ERROR_</title>
     <link
       rel="stylesheet"
-      href="/resources/dist/css/access_confirmation.bundle.css"
+      href="/dist/css/access_confirmation.bundle.css"
```

Comment:
Should this be `th:href`?

Reply:
Yes, and yes.  Both fixed in 5b3276911b8bf204112ff5bcb359b0e27e547c4c